src/vm/docs/mod.rs

```diff
@@ -643,6 +643,17 @@ the tuple. If the supplied option is a `(none)` option, get returns `(none)`.",
 "
 };
 
+const TUPLE_SET_API: SpecialAPI = SpecialAPI {
+    input_type: "tuple, tuple",
+    output_type: "tuple",
+    signature: "(set tuple { key1: val1 })",
+    description: "The `set` function returns a new tuple with the combined fields, without mutating the supplied tuples.",
+    example: "(define-map users { id: int } { name: (string-ascii 12), address: (optional principal) })
+(map-insert users { id: 1337 } { name: \"john\", address: none }) ;; Returns true
+(let ((user (unwrap-panic (map-get? users { id: 1337 }))))
+    (set user { address: (some 'SPAXYA5XS51713FDTQ8H94EJ4V579CXMTRNBZKSF) })) ;; Returns (tuple (address (some SPAXYA5XS51713FDTQ8H94EJ4V579CXMTRNBZKSF)) (name \"john\"))"
+};
```

Comment:
Can this example be simplified to focus more on the documented function rather than map-functionality?